protobuf2.5/plan.sh

```diff
@@ -0,0 +1,12 @@
+pkg_name=protobuf
+pkg_origin=core
+pkg_version=2.5.0
+pkg_license=('BSD-2-Clause')
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_source=https://github.com/google/${pkg_name}/releases/download/v${pkg_version}/${pkg_name}-${pkg_version}.tar.bz2
```

Comment:
protobuf appears to be released under `BSD-3-Clause`: https://github.com/google/protobuf/blob/v2.5.0/COPYING.txt